plugins/gatsby-source-bxjs/package.json

```diff
@@ -7,5 +7,11 @@
   },
   "keywords": [],
   "author": "",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
those deps are installed and used from top level, not really needed here

Reply:
Let it be independent

Reply:
yarn installs them anyway. So later if you drop that plugin - no left owers

Reply:
But we can't really drop it though